package-lock.json

```diff
@@ -1,8808 +1,8 @@
 {
   "name": "llaminator",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
I think this might be part of the reason you've needed to do this twice now. This indicates that you're using an older version of npm (see https://github.com/GoogleChromeLabs/llaminator/issues/34#issuecomment-991802512). I'm on npm 8.1.2 (the version that got installed when I ran `nvm install node`.

Reply:
Oh you're right, thanks!